### DISCUSSION TITLE
Could not create share link

### DISCUSSION THREAD

**Discussion by primer21** [mikumifa/biliTickerBuy#149] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/173460147/67422d05-f6fa-461d-b25e-c20d490a037d)
请问这个问题有影响吗，网上找不到解决方法

**Comment by primer21** [comment]
![image](https://github.com/mikumifa/biliTickerBuy/assets/173460147/db1c241c-b428-4a2d-abc6-a4a4dafeb38a)
cqi重启后是这样

**Reply by mikumifa** [reply]
无影响，正常访问网站

**Reply by primer21** [reply]
作者大大，再询问一下为什么点这个获取票消息没反应，app里也没内容呀
![Uploading image.png…]()

**Reply by mikumifa** [reply]
看一下说明书

**Reply by primer21** [reply]
可是我已经读过很多遍了还是没发现问题所在，已经确保手机上地址和购票人填了，还是点购票信息没反应
![image](https://github.com/mikumifa/biliTickerBuy/assets/173460147/2deee07a-f8b0-407b-832e-c78ff58a017e)
![Uploading image.png…]()

**Reply by primer21** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/173460147/c5aa0558-6e24-4efd-bd40-0166e9079d66)

**Reply by mikumifa** [reply]
有过弹出浏览器吗，然后登录吗

**Reply by mikumifa** [reply]
https://github.com/mikumifa/biliTickerBuy/wiki/%E6%B5%8F%E8%A7%88%E5%99%A8%E9%A9%B1%E5%8A%A8%E5%AE%89%E8%A3%85%E5%87%BA%E9%97%AE%E9%A2%98%E7%9A%84%E8%A7%A3%E5%86%B3%E6%96%B9%E5%BC%8F

**Reply by mikumifa** [reply]
你的电脑系统有点老，应该不是最新的win10或者win11

**Reply by primer21** [reply]
没有弹出过，我也在思考这个问题是怎么识别我的cookie的我还以为是那个链接里面包含了我的cookies消息，我电脑是win11版本好像还没注意，但是之前用seleium也可以启动edge，但是这个没有弹出过浏览器

**Reply by primer21** [reply]
是驱动问题吗，是的话我换这个方向试试看

**Reply by primer21** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/173460147/9ce7a8f7-84ea-47b6-b585-f8f956ffb92a)

**Reply by mikumifa** [reply]
不用管驱动了，按照上面的网址 生成cookie.json直接导入就行

**Reply by mikumifa** [reply]
专业版？
edge最新？

**Reply by primer21** [reply]
正式版，是最新

**Reply by mikumifa** [reply]
那未知了

**Comment by primer21** [comment]
没有弹出过，我也在思考这个问题是怎么识别我的cookie的我还以为是那个链接里面包含了我的cookies消息，我电脑是win11版本好像还没注意，但是之前用seleium也可以启动edge，但是这个没有弹出过浏览器


&nbsp;


愿为星星候着你^ω^
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年6月21日(星期五) 晚上9:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mikumifa/biliTickerBuy] Could not create share link (Discussion #149)














 
有过弹出浏览器吗，然后登录吗
